### PR TITLE
Allow updating the release date

### PR DESCRIPTION
**Pull request by pakrym**
To be used in the Prepare-Release script.

### CODE REVIEW

File: eng/common/Update-Change-Log.ps1 [hunk 1]
```diff
@@ -12,7 +12,8 @@ param (
   [Parameter(Mandatory = $true)]
   [String]$ChangeLogPath,
   [String]$Unreleased = $True,
-  [String]$ReplaceVersion = $False
+  [String]$ReplaceVersion = $False,
+  $ReleaseDate
```
Comment (weshaggard) [inline eng/common/Update-Change-Log.ps1:16]: What is the type for this?

Comment (weshaggard) [inline eng/common/Update-Change-Log.ps1:16]: What is the type for this?

Comment (pakrym) [reply eng/common/Update-Change-Log.ps1]: Fixed

File: eng/common/Update-Change-Log.ps1 [hunk 2]
```diff
@@ -95,7 +100,7 @@ function Get-NewChangeLog( [System.Collections.ArrayList]$ChangelogLines, $Versi
       exit(0)
    }
 
-   if (($ReplaceVersion -eq $True) -and ($Unreleased -eq $False) -and $CurrentTitle.Contains($version) -and (-not $CurrentTitle.Contains($UNRELEASED_TAG))) {
+   if (($ReplaceVersion -eq $True) -and ($Unreleased -eq $False) -and $CurrentTitle.Contains($version) -and (-not $CurrentTitle.Contains($UNRELEASED_TAG)) -and (-not $ReleaseDate)) {
```
Comment (weshaggard) [inline eng/common/Update-Change-Log.ps1:103]: I must say I'm have a hard time understanding this condition :)

Why did you need to add the ReleaseTag option to it? Is it because you are trying to change the release date?

Comment (pakrym) [reply eng/common/Update-Change-Log.ps1:103]: This condition prevents the changelog line from being updated when the only difference is the date. I want to be able to change the date of the existing entry that is not `unreleased`.

**Review by weshaggard — APPROVED** [review]
